_data/accessibility-tests/radio-buttons.yml

```diff
@@ -0,0 +1,72 @@
+title: Radio buttons
+component_status: pass
+test_items:
+# General tests
+  - summary: Focus indicator on radio button is clearly visible.
+    summary_additional: When you use a keyboard to focus on a radio button group, you will see a visible outline or other clear indication of focus.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+  - summary: Radio button labels are descriptive, brief and clear.
+    summary_additional: When you view radio buttons, all labels have short descriptions that are easy to understand and act on.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.6
+  - summary: All clickable elements touch target size meet height and width requirements.
+    summary_additional: The size and spacing of the touch target are at least 24px wide and 24px tall.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.8
+# Keyboard navigation tests
+  - summary: Radio button focus can be moved and selection can be selected or deselected.
+    summary_additional: When you use the spacebar and arrow keys on a keyboard, you can toggle between selections and clearly see when the choice is selected or not.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+  - summary: Radio button focus follows logical content order.
+    summary_additional: When you tab through the radio button group with a keyboard, the focus stays in order of the content as intended.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.3
+  - summary: Radio buttons are still fully functional using only keystrokes.
+    summary_additional: When you use a keyboard to navigate to the radio button option, you are able to move focus, check and uncheck buttons, and move focus into and out of the radio group. If none of the radio buttons are checked, focus is set on the first radio button in the group.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+# Screen reader tests
+  - summary: Screen reader announces question before any radio button and then related elements.
+    summary_additional: When listening to the radio button using a screen reader, you hear the question related to the radio button read first. Then you will hear the radio button and label and its checked state.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.2
+    wcag_criterion: 4.1.2
+  - summary: Screen reader announces instructions for radio button fields and options for radio button groups.
+    summary_additional: When listening to the radio button using a screen reader, any instructions related to the field and you also hear all the options in the button group.
+    test_status: pass
+    test_type: screen_reader
```

Comment:
@amycole501 @alex-hull,

**question:** This is marked as a general test on the [checkbox page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/rc-publish-checkbox-checklist/components/checkbox/accessibility-tests/), but a screen reader test here. It also seems to be mixed test types on other component pages ([text input](https://designsystem.digital.gov/components/text-input/accessibility-tests/) | [search](https://designsystem.digital.gov/components/search/accessibility-tests/) ). Should the test type match across pages? (At this time, I'm mostly concerned with radio and checkbox)

Reply:
3.3.2 is General, yes. It's only if the instructions are visible.

Reply:
No changes made for this comment.

Reply:
@amycole501 Did you mean that 3.3.2 should be changed to a general test for radio, or did you want to leave it as a screen reader test on this page?

Reply:
It can be a general test for radio. It shouldn't be a screen reader test for any component.

Reply:
@RachelCorsino Can you move this to a general test? If we draw from the summary from the [checkbox component](https://github.com/uswds/uswds-site/pull/2803), we can re-write the summary to something like:

> summary: Visible instructions explain radio button requirements. 
    summary_additional: Each radio button group has a visible label above it to indicate its purpose. For example, "Select your state."
    test_type: general

Reply:
Resolved in [503781a](https://github.com/uswds/uswds-site/pull/2821/commits/503781a1a88085a08f85d5a5eaa4e3ab22bb71e6)